ci/ray_ci/tests.env.Dockerfile

```diff
@@ -8,6 +8,8 @@ ARG BUILD_TYPE
 ENV CC=clang
 ENV CXX=clang++-12
 
+SHELL ["/bin/bash", "-ice"]
```

Comment:
this is required to use oss-ci-base_build for testing

Reply:
why? what is the error?

rayci already sets shell to `-elic` for the docker plugin?

Reply:
yeah -elic is only provided during docker run; this is docker build; the oss_ci_base_build is already have pip installed (through ci/env/install-dependencies.sh i assume), but without this the docker build process will say pip not found (assume pip is loaded through .bashrc)

Reply:
the error: https://buildkite.com/ray-project/premerge/builds/9322#018b4e69-c8d5-4bdb-ad01-ac0a9910630c/6-99

Reply:
Keeping only `-i` sounds good, I want to load the .bashrc instead of only getting pip, there are quite a few places setting things up in .bashrc (https://github.com/ray-project/ray/blob/master/.buildkite/pipeline.ml.yml#L367). 

A few things at put in root/.bashrc, others are in ~/.bashrc, so `-i` will be a safer option to set all things up.